From 6d4307ae8cb103a83f097e80592449c55513d282 Mon Sep 17 00:00:00 2001
From: liushuyu <liushuyu011@gmail.com>
Date: Tue, 21 Feb 2017 22:02:55 -0700
Subject: [PATCH] Docker Systemd service file tuning

* Make Docker uses higher-performance overlayfs2 module by default
* Unlimited tasks
---
 contrib/init/systemd/docker.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/init/systemd/docker.service b/contrib/init/systemd/docker.service
index 5174631..ae85d55 100644
--- a/contrib/init/systemd/docker.service
+++ b/contrib/init/systemd/docker.service
@@ -10,7 +10,7 @@ Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd -H fd://
+ExecStart=/usr/bin/dockerd --storage-driver=overlay2 -H fd://
 ExecReload=/bin/kill -s HUP $MAINPID
 LimitNOFILE=1048576
 # Having non-zero Limit*s causes performance problems due to accounting overhead
@@ -19,7 +19,7 @@ LimitNPROC=infinity
 LimitCORE=infinity
 # Uncomment TasksMax if your systemd version supports it.
 # Only systemd 226 and above support this version.
-#TasksMax=infinity
+TasksMax=infinity
 TimeoutStartSec=0
 # set delegate yes so that systemd does not reset the cgroups of docker containers
 Delegate=yes
-- 
2.11.1

